### PR TITLE
Added active state to menu based on the page you are on

### LINKED ISSUE #83: Menu item is not highlighted when active
## Summary
[list the steps to replicate the issue]
Going on a secondary page, doesn't highlight the active item in the menu

## Current and expected result
The active item should be highlighted in the menu
e.g. OpenStack page, the item in the menu should have the Vanilla underline below?

## Screenshot
52205384-cf3e7000-286e-11e9-9b8e-c45858f0f965

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

Added active state to menu based on the page you are on

## QA

1. download this branch
2. run the site and go to any page other than the homepage
3. see that the nav has an active state with an orange line below the menu item you are on

## Issue / Card

Fixes #83

## Screenshots

image

### CODE REVIEW

**Comment by clagom** [comment]
screenshot 2019-02-04 at 13 06 10
I see this overlapping now, it wasn't like this in the previous one

File: templates/_base/base.html [hunk 1]
```diff
@@ -63,37 +63,40 @@
...
+            <a href="/download" {% if level_1 == 'download' %}class="active"{% endif %}>ダウンロード</a>
           </li>
         </ul>
       </nav>
     </div>
   </header>
 
   {% block header %}{% endblock %}
-
+<div id="main-content" class="u-no-margin--top">
```
Comment (anthonydillon) [inline templates/_base/base.html]: All the pages have main-content div in them so you'll either need to remove them or this div. Also, no margin top should not be needed.

Comment (pmahnke) [reply templates/_base/base.html]: @anthonydillon 	I remove the main-content div from all the main pages... I have rebased now.

**Comment by anasereijo** [comment]
Looks nice 🥇

**Pull request merged**